libretro-fetch.sh not downloading all the cores

When I try to get all the cores with the command "./libretro-fetch.sh" or "make", it stops at Flycast:

```
=== Flycast
libretro_fetch:No URL set to fetch Flycast via git.
Makefile:7: recipe for target 'fetch' failed
make: *** [fetch] Error 1
```
Only these cores are properly downloaded: mednafen_snes, snes9x, bsnes_accuracy, snes9x2002, bsnes_balanced, snes9x2005, bsnes_cplusplus98, snes9x2010, bsnes_mercury_accuracy, bsnes_mercury_balanced, bsnes_mercury_performance and bsnes_performance. 

Everything seems to work fine until Flycast, then no more cores are downloaded.

I could download them all with no problems a few weeks ago. This must be a recent change; perhaps it's got something to do with "Renamed to flycast", but I don't really know.

--- a/recipes/linux/cores-linux-x86-generic
+++ b/recipes/linux/cores-linux-x86-generic
@@ -89,7 +89,7 @@ puae libretro-uae https://github.com/libretro/libretro-uae.git master YES GENERI
 px68k libretro-px68k https://github.com/libretro/px68k-libretro.git master YES GENERIC Makefile.libretro .
 quasi88 libretro-quasi88 https://github.com/libretro/quasi88-libretro.git master YES GENERIC Makefile .
 quicknes libretro-quicknes https://github.com/libretro/QuickNES_Core.git master YES GENERIC Makefile .
-flycast libretro-flycat https://github.com/libretro/flycast.git master YES GENERIC Makefile . HAVE_OIT=1
+flycast libretro-flycast https://github.com/libretro/flycast.git master YES GENERIC Makefile . HAVE_OIT=1
 reminiscence libretro-reminiscence https://github.com/libretro/REminiscence.git master YES GENERIC Makefile .
 remotejoy libretro-remotejoy https://github.com/libretro/libretro-remotejoy.git master YES GENERIC Makefile .
 sameboy libretro-sameboy https://github.com/libretro/SameBoy.git buildbot YES GENERIC Makefile libretro

--- a/rules.d/core-rules.sh
+++ b/rules.d/core-rules.sh
@@ -165,7 +165,7 @@ libretro_snes9x_git_url="https://github.com/libretro/snes9x.git"
 libretro_snes9x_build_subdir="libretro"
 
 include_core_flycast() {
-	register_module core "Flycast"
+	register_module core "flycast"
 }
 libretro_flycast_name="Flycast"
 libretro_flycast_git_url="https://github.com/libretro/flycast.git"

